### PR TITLE
Feature branch : HTML/CSS & JavaScript capstone project 

### PR DESCRIPTION
**Pull request by abenezer-addisu**
In This Branch :  
I create a conference website with two page that show about the conference speaker and partners. includes :-

- [x] Responsive Website for all screens

- [x] CSS grid for the layout structuring

- [x] Flexbox for structuring instead of floats

- [x] JavaScript for interactive

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,200 @@
...
+- **Javascript for interactive**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+
```
Comment (AbelG101) [inline README.md]: - [ ] Hey Abenezer, please make sure to deploy the project and add the link to your readme on the "Live Demo" section as this is part of the project requirements.
- Here's the requirement as a reminder: 👇
   image
- [ ] As part of the project requirements please make sure to record a video describing your project (not longer or shorter than 5mins). You can attach it in the readme or in the PR description.
  image

**Comment by AbelG101** [comment]
- Hello Abenezer, please restrain from adding features once you've requested a review as this could lead to invalidation.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+        <li><img src="assets/images/OER Live Logo FINAL.webp" alt="Roke" /></li>
+        <li><img src="assets/images/gitex_logo_png-min.png" alt="Roke" /></li>
+      </ul>
+    </section>
+    <footer class="footer">
+      <!-- <a href="./index.html"> <img class="footer_logo" src="assets/images/GITEX+Africa+Website+Logo.jpg" alt="gitex logo" > </a> -->
+      <div class="rights">
+        <p>2023 Abenezer Addisu, All rights are Reserved</p>
+          <p>Built by ❤️ Abenezer</p>
+      </div>
```
Comment (AbelG101) [inline index.html:169]: **I just want to say your project is looking 👌, great job!**
- [ ] Ouch! this is close but as part of the design specification you need to add a logo to the left of the text in your footer
- Behance design:
  image
- Your design:
  image

- [ ] This applies to the about page as well. ✔

File: src/css/top-section.css [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+}
+
+.mobileLinkText {
+  text-decoration: none;
+  color: #fff5e1;
+  font-family: 'Inter', serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+}
```
Comment (AbelG101) [inline src/css/top-section.css]: - [ ] I like that you're trying to add your own flavor 🧂 here but it's best to stay consistent in your design and use the given font from the design (Cocogoose, Lato). 
- [ ] You can just remove the `font-family` property at line `59`

File: index.html [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+  </head>
+  <body>
+    <section class="header">
+      <header>
+        <nav class="nav">
+          <div class="mobile-menu">
+            <i class="fa-sharp fa-solid fa-bars"></i>
+          </div>
+          <div class="menu-mobile hideMenu">
+            <button class="close"><h2>X</h2></button>
```
Comment (AbelG101) [inline index.html:27]: - [ ] [OPTIONAL] There's something funny going on with the close menu button here please check the video below 👇
  firefox_vd9TroorKi
- Please make sure you improve it as it leads to a bad UX

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @abenezer-addisu 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Unique content ✔
-  UI is looking on point ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- You just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- Check the last comment...

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (tsohleDev) [inline index.html]: - [ ] I suggest you include the logo, as the previous reviewer also requested, as it is very much important to be able to implement the UI design as it is, as a front-end developer. 👨‍💻 Note that you could only change the data, not the design, which is something you will do a lot as a **front end developer** 😉

File: index.html [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+          <div class="date-location">
+            <h2>2023.02.30(WED) ~ 15(FRI)</h2>
+              <h3>@ millnium Hall , Addis Ababa</h3>
+          </div>
+        </div>
+        <img
+          class="friends"
+          src="assets/images/conferencethemenewImg.jpg"
+          alt="gitex conference theme."
+        />
```
Comment (tsohleDev) [inline index.html:100]: [COMPLIMENT] I Really love the animation you added here, it gives the page a nice feel 💯 💯 🥇

File: src/css/top-section-about.css [hunk 6]
```diff
@@ -0,0 +1,500 @@
...
+  color: #6f6c6b;
+  border: none;
+  transform: rotate(180deg);
+  scale: 1.5;
+}
+
+.mobileLinkText {
+  text-decoration: none;
+  color: #fff5e1;
+  font-family: 'Inter', serif;
```
Comment (tsohleDev) [inline src/css/top-section-about.css]: - [ ] I think its better to stick to the design specifications, and use the font families that are defined there, as this makes your page consistent, and also shows that you are able to utilize a design template as a **front end developer** 😉

File: index.html [hunk 7]
```diff
@@ -0,0 +1,174 @@
...
+          </p>
+        </li>
+      </ul>
+      <button class="get-ticket" type="button">Get your ticket Here</button>
+    </section>
+
+    <section class="guests" id="guests">
+      <h2>SPEAKERS</h2>
+      <div class="guests-list"></div>
+    </section>
```
Comment (tsohleDev) [inline index.html:151]: [COMPLIMENT] Great work on the ``More`` button in the **speakers** ``section`` 💯 💯 🔥

**Review by tsohleDev — CHANGES_REQUESTED** [review]
## Hi Micronaut @abenezer-addisu 

Good job so far!
There are some issues that you still need to work on.

163947610-e4d849d5-e69c-49b9-90bd-a4191f4e2856

## Highlights!
- Live Demo ✔️ 
- RWD ✔️ 
- Followed Design ❌

Hello ✋, you have done well creating your project 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## Compliment
_Every comment with the [COMPLIMENT] prefix is not  a request, just a compliment at your skills_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@tsohleDev` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/css/top-section-about.css (hunk 6 above)
Comment (abenezer-addisu) [reply src/css/top-section-about.css]: @tsohleDev  Can you please provide me with more brief information about this comment?

File: src/css/top-section.css [hunk 8]
```diff
@@ -56,7 +56,6 @@
 .mobileLinkText {
   text-decoration: none;
   color: #fff5e1;
-  font-family: 'Inter', serif;
```
Comment (Abenezer-Tilahun) [inline src/css/top-section.css]: - [ Optional ] As stated by the previous reviewer Kindly use` font-family: 'Cocogoose', sans-serif;`or `font-family: 'Lato', sans-serif;` to stay consistent in your design 👍 .

File: index.html (hunk 2 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @abenezer-addisu 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub -Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.
✔️ Beautiful Design 👍 💯 

## Optional suggestions 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**